tests/unit/test_cli.py

```diff
@@ -170,8 +170,8 @@ def test_loglevel_option(
         self,
         loglevel_args: list[str],
         expected_level: str,
-        tmp_cwd,
-    ):
+        tmp_cwd: Iterator[Path],
```

Comment:
```suggestion
        tmp_cwd: Path,
```
pytest does some magic so `tmp_cwd` behaves like a regular `Path` in test functions (even though the fixture function returns an iterator / is a context manager)